test/common/testlib.py

```diff
@@ -93,6 +93,21 @@
 opts.jobs = 1
 opts.fetch = True
 
+default_layouts = [
+    {
+        "name": "desktop",
+        "is_mobile": False,
```

Comment:
This just describes whether cockpit's shell switches to desktop or mobile mode in the given resolution, right? So this is not an independent variable, but depends on the window width. Is this too complicated to determine automatically? Please add a comment.

But see below, I'm fairly confident that we can lose this.

Reply:
> This just describes whether cockpit's shell switches to desktop or mobile mode in the given resolution, right?

Yes.  It's for the benefit of a few utility functions only.  But since tests now know at every point in which layout they are in, I think we don't need this at all anymore.  It's buggy right now, as you noticed, and the tests are fine.

So I'll remove `is_mobile`.